content/run.md

```diff
@@ -154,27 +154,27 @@ First, you need to change the global input data directory variable inside the sc
 
     dir_inputdata='/cluster/shared/noresm/inputdata/'
     
-Choose a location for storing the domain and surface data files for Scandinavia. Set the environment variable 
-for the directory as below:
+Choose a location for storing the domain and surface data files for Scandinavia (eg. /cluste/projects/nn2806k/$USER/regiondata). 
+Set the environment variable for the directory as below:
 
-    REGDATA='/cluste/projects/nn2806k/$USER/'
+    MYREGDATA='/cluste/projects/nn2806k/$USER/regiondata/'
 
-```{discussion} Choose what resolution global surface data you like to use? 
-For example: for 0.5x0.5 resolution, choose /cluster/shared/noresm/inputdata/lnd/clm2/surfdata_map/surfdata_360x720cru_16pfts_Irrig_CMIP6_simyr2000_c170824.nc
+```{discussion} Choose the global surface data file corresponding to the resolution! 
+For example: for 0.5x0.5 resolution, choose surfdata_360x720cru_16pfts_Irrig_CMIP6_simyr2000_c170824.nc (under shared noresm folder)
```

Comment:
put `surfdata_360x720cru_16pfts_Irrig_CMIP6_simyr2000_c170824.nc` to indicate that it is a file